runtime/src/main/java/org/corfudb/runtime/CorfuRuntime.java

```diff
@@ -33,6 +50,8 @@
 import org.corfudb.runtime.exceptions.WrongClusterException;
 import org.corfudb.runtime.exceptions.unrecoverable.UnrecoverableCorfuError;
 import org.corfudb.runtime.exceptions.unrecoverable.UnrecoverableCorfuInterruptedError;
+import org.corfudb.runtime.object.CorfuCompileProxy;
+import org.corfudb.runtime.object.ICorfuSMR;
```

Comment:
push this down to the ObjectsView class, this class doesn't need to know about these types.

Reply:
Done.